datasets/pib/pib.py

```diff
@@ -0,0 +1,169 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CVIT IIIT-H PIB Multilingual Corpus"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{cvit-pib:multilingual-corpus,
+title = {Revisiting Low Resource Status of Indian Languages in Machine Translation},
+authors={Jerin Philip, Shashank Siripragada, Vinay P. Namboodiri, C.V. Jawahar
+},
+year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+This new dataset is the large scale sentence aligned corpus in 11 Indian languages, 
+viz. CVIT-PIB corpus that is the largest multilingual corpus available for Indian languages.
+"""
+
+_URL = "http://preon.iiit.ac.in/~jerin/resources/datasets/pib-v0.tar"
+
+_LanguagePairs = [
+    "or-ur",
+    "ml-or",
+    "bn-ta",
+    "gu-mr",
+    "hi-or",
+    "en-or",
+    "mr-ur",
+    "en-ta",
+    "hi-ta",
+    "bn-en",
+    "bn-or",
+    "ml-ta",
+    "gu-ur",
+    "bn-ml",
+    "ml-pa",
+    "en-pa",
+    "bn-hi",
+    "hi-pa",
+    "gu-te",
+    "pa-ta",
+    "hi-ml",
+    "or-te",
+    "en-ml",
+    "en-hi",
+    "bn-pa",
+    "mr-te",
+    "mr-pa",
+    "bn-te",
+    "gu-hi",
+    "ta-ur",
+    "te-ur",
+    "or-pa",
+    "gu-ml",
+    "gu-pa",
+    "hi-te",
+    "en-te",
+    "ml-te",
+    "pa-ur",
+    "hi-ur",
+    "mr-or",
+    "en-ur",
+    "ml-ur",
+    "bn-mr",
+    "gu-ta",
+    "pa-te",
+    "bn-gu",
+    "bn-ur",
+    "ml-mr",
+    "or-ta",
+    "ta-te",
+    "gu-or",
+    "en-gu",
+    "hi-mr",
+    "mr-ta",
+    "en-mr",
+]
+
+
+class PibConfig(datasets.BuilderConfig):
+    """BuilderConfig for PIB"""
+
+    def __init__(self, language_pair, **kwargs):
+        super().__init__(**kwargs)
+        """
+
+        Args:
+            language_pair: language pair, you want to load
+            **kwargs: keyword arguments forwarded to super.
+        """
+        self.language_pair = language_pair
+
+
+class Pib(datasets.GeneratorBasedBuilder):
+    """This new dataset is the large scale sentence aligned corpus in 11 Indian languages, viz.
+    CVIT-PIB corpus that is the largest multilingual corpus available for Indian languages.
+    """
+
+    BUILDER_CONFIG_CLASS = PibConfig
+    BUILDER_CONFIGS = [PibConfig(name=pair, description=_DESCRIPTION, language_pair=pair) for pair in _LanguagePairs]
+
+    def _info(self):
+        # TODO: Specifies the datasets.DatasetInfo object
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION + f" Available language pairs are {_LanguagePairs}",
+            # This defines the different columns of the dataset and their types
+            features=datasets.Features(
+                {
+                    "src_lang": datasets.Value("string"),
+                    "tgt_lang": datasets.Value("string")
```

Comment:
Can you use the Translation feature type instead ?
You can find an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py)